app/presenters/curation_concerns/presents_attributes.rb

```diff
@@ -39,11 +40,24 @@ def microdata_type_to_html
 
     private
 
-      def renderer_for(field, options)
-        if options[:catalog_search_link]
-          Renderers::LinkedAttributeRenderer
-        elsif field == :rights
-          Renderers::RightsAttributeRenderer
+      def find_renderer_class(name)
```

Comment:
Can we use a short-circuit pattern here instead of the loop break and keeping track of whether `renderer` is `nil` or not?

```ruby
def find_renderer_class(name)
    [ 'Renderer', 'AttributeRenderer'].each do |base|
    const_name = "#{name.to_s.camelize}#{base}"
    return Renderers.const_get(const_name) if Renderers.const_defined?(const_name)
  end
  raise NameError, "unknown renderer type `#{name}`"
end
```